src/otx/algo/object_detection_3d/losses/ddn_loss.py

```diff
@@ -0,0 +1,251 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""ddn loss for MonoDETR model."""
+from __future__ import annotations
+
+import math
+
+import torch
+from torch import nn
+
+from otx.algo.common.losses.focal_loss import FocalLoss
+
+
+def compute_fg_mask(
+    gt_boxes2d: torch.Tensor,
+    shape: tuple[int, int],
+    num_gt_per_img: int,
+    downsample_factor: int = 1,
+    device: torch.device | None = None,
+) -> torch.Tensor:
+    """Compute foreground mask for images.
+
+    Args:
+        gt_boxes2d [torch.Tensor(B, N, 4)]: 2D box labels
```

Comment:
```suggestion
        gt_boxes2d (torch.Tensor(B, N, 4)): 2D box labels
```
Could you update others also to align it to a style OTX uses?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/4031/commits/00200f4d4811d82e86e371cb5dabd2541098e83c